---
permalink: /
title: "Ghost Knockoffs software: an enhanced GWAS approach"
excerpt: "About me"
author_profile: true
redirect_from: 
  - /about/
  - /about.html
---

Recent advances in genome sequencing and imputation technologies provide an exciting opportunity to study the contribution of genetic variants to disease phenotypes, but our ability to explain heritability and to translate genetic risk into mechanistic insight remains limited at this point. In our recent paper that can be found [here](https://google.com), we propose a multiple knockoffs inference to enhance the discovery of genome-wide association studies (GWAS), by improving its statistical power for locus discovery and its resolution at each locus to prioritize the putative causal variants. 

Our Software nicknamed **"GhostKnockoff"** is free to use and publicly available. The [GitHub](https://sosaar@github.com) has all pertinent code and dependencies. 

About
======
We develop a method to perform knockoff-based inference without generating any individual-level knockoffs. The method requires only summary statistics (Z-score) from conventional GWAS and retains useful features of knockoff-based inference. The method additionally allows meta-analysis of studies with arbitrary sample overlapping. It is also robust to genetic variants that do not show up in all studies, and therefore applicable to the integration of genetic association studies genotyped using SNP-array, whole exome-sequencing and whole genome sequencing. 

Getting started
======
1. First, 

Tips
------


Terms
======
When using GhostKnockoff inference in your work, please cite our [flagship paper](https://nature.com)

Communication
=====
Don't hesitate to reach out to our research group using the following emails: zihuai@stanford.edu, sosaar@stanford.edu
